docs/podman-create.1.md

```diff
@@ -654,6 +654,40 @@ change propagation properties of source mount. Say `/` is source mount for
 To disable automatic copying of data from the container path to the volume, use
 the `nocopy` flag. The `nocopy` flag can be set on bind mounts and named volumes.
 
+**--volumes-from**[=*[CONTAINER[:OPTIONS]]*]
```

Comment:
If you include the literal `=`, you need to have *CONTAINER*, right?  So I think this should be:

```markdown
**--volumes-from**[=*CONTAINER*[:*OPTIONS*]]
```

Reply:
Fixed